.travis.yml

```diff
@@ -7,11 +7,15 @@ sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bower's cache
 
 env:
+  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
   - EMBER_TRY_SCENARIO=default
```

Comment:
Personally I think that it'd be better to optimize for comprehensive test coverage rather than test run time in the CI environment. Within the next 6 months we'll have another LTS version to support and I don't think we would want to make a sacrifice then as well to try to maintain a 5-test maximum limit.

Also, the fact that we have canary test coverage has saved me from spending time debugging when unnecessary, so it seems like a time-saver to me as a maintainer.

https://github.com/sir-dunxalot/ember-tooltips/issues/129